### PR TITLE
Add grafana dashboard for heptio ark

### PR DESCRIPTION
**Pull request by mwieczorek**
**Description**

Changes proposed in this pull request:

- Add grafana dashboard for heptio ark
- fix ark's service monitor (proper namespace)

**Related issue(s)**

### CODE REVIEW

**Comment by mwieczorek** [comment]
image

example dashboard

File: resources/ark/templates/service-monitor.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
   name: {{ template "ark.fullname" . }}
+  namespace: kyma-system
```
Comment (venturasr) [inline resources/ark/templates/service-monitor.yaml]: `namespace:` should be defined as a template. Also can be removed as `helm `take care of the namespaces.

Comment (mwieczorek) [reply resources/ark/templates/service-monitor.yaml]: ark is installed in a different namespace than Prometheus operator so I need to put the namespace for the service monitor explicitly. Without it the service monitor won't be picked up by the operator.
I extracted the value to the values.yaml

**Comment by mwieczorek** [comment]
I've also made some changes to the dashboard. New layout:

image

**Pull request merged**